sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -74,7 +74,7 @@ class RunningCountStatefulProcessorWithTTL
       outputMode: OutputMode,
       timeMode: TimeMode): Unit = {
     _countState = getHandle.getValueState[Long]("countState",
-      Encoders.scalaLong, TTLConfig(Duration.ofMillis(1000)))
+      Encoders.scalaLong, TTLConfig(Duration.ofMillis(30000)))
```

Comment:
It's a huge change of the value of TTL - if the existing test does not fail with this change, it sounds to me as the existing test is too loose.

Reply:
Yea - actually that test was not relying on these configs at all. But anyway - just moved it to the data source related suite as a separate processor.